src/Parser/RetributionPaladin/Modules/Items/Tier20_2set.js

```diff
@@ -48,17 +48,16 @@ class Tier20_2set extends Module {
 		};
 	}
 
-	//TODO add suggestion later 
-	// suggestions(when) {
-	// 	when(this.percentUptime).isLessThan(.95)
-	// 		.addSuggestion((suggest, actual, recommended) => {
-	// 			return suggest(`Your Tier 20 2pc uptime of ${formatPercentage(actual)}% is below 95%, try to use judgement as much as possible`)
-	// 				.icon(SPELLS.RET_PALADIN_T20_2SET_BONUS_BUFF.id)
-	// 				.actual(`${formatPercentage(actual)}% uptime`)
-	// 				.recommended(`${formatPercentage(recommended)}% is recommended`)
-	// 				.regular(recommended).major(recommended - 0.05);
-	// 		});
-	// }
+	suggestions(when) {
+	 	when(this.percentUptime).isLessThan(.95)
+	 		.addSuggestion((suggest, actual, recommended) => {
+ 				return suggest(`Your Tier 20 2pc uptime of ${formatPercentage(actual)}% is below 95%, try to use judgement as much as possible`)
```

Comment:
When reviewing a log would you really look at the tier 20 2 set uptime and tell someone this? I imagine you'd look if someone has the 2PT20 and if so you would look at his Judgment casts, and if this is too low tell him to cast Judgment more. In that case I would recommend moving this suggestion to Judgment in the CastEfficiency module; you should be able to set it as an `extraSuggestion` with `isActive` toggling between default and 2PT20 equipped checks.